src/K9OCRS/ClientApp/src/areas/applications/Confirmation/index.js

```diff
@@ -1,74 +1,98 @@
-import React from "react";
+import axios from "axios";
+import React, { useState, useEffect } from "react";
+import { connect } from "react-redux";
+import { useParams } from "react-router-dom";
+import selectors from "../../../shared/modules/selectors";
 
-const Confirm = () => {
+
+const Confirm = props => {
+
+  const { sectionId } = useParams();
+  const [sectionDetail, setSectionDetail] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [alerts, setAlerts] = useState([]);
+  const { currentUser } = props;
+
+  useEffect(() => {
+      async function getTest() {
+        if (sectionId) {
+          try {
+            const res = await axios.get(`/api/ClassSections/ ${sectionId}`);
+            setSectionDetail(res?.data);
+            setLoading(false);
+          } catch(err) {
+            setLoading(false);
+            setAlerts([{ color: 'danger', message: 'We\'re having issues getting the details for this class' }]);
+          }
+        }
+      }
+      getTest();
+    }, [sectionId]);
+
+    let x = sectionDetail?.rosterCapacity;
+    let y = sectionDetail?.rosterActual;
+    let z = x-y;
+    
   return (
     <div className="container">
-      <h3>Thank you for choosing "selected class:"</h3>
+      <h3>Thank you for choosing the following class:</h3>
       <div className="mt-3 mb-3">
         <div className="card">
           <div className="card-header">
-            <h3>STAR Puppy</h3>
+            <h3>{sectionDetail?.classType?.title}</h3>
           </div>
           <div className="card-body">
-            Raising a puppy can be fun and very rewarding and also a big
-            challenge! The K-9 Obedience Club offers classes to help you train
-            your puppy to become a self-confident, happy and easy-to-live-with
-            companion.
+          {sectionDetail?.classType?.description}
           </div>
           <div className="card-footer">
             <span className="text-center font-weight-bold">
-              Section: SP01 | Day: Monday | Time: 6PM | Start: 12/20/21 |
-              Duration: 7 weeks | Instructor: Rebecca Grinsell | Seats: 5
+              Section: {sectionDetail?.id} | Day: Monday | Time: 6PM | Start: {sectionDetail?.startDate} |
+              Duration: 7 weeks | Instructor: {sectionDetail?.instructor?.firstName} {sectionDetail?.instructor?.lastName} | Available Seats: {z}
```

Comment:
This completes the change from the comment above
```suggestion
              Duration: 7 weeks | Instructor: {sectionDetail?.instructor?.firstName} {sectionDetail?.instructor?.lastName} | Available Seats: {availableSeats}
```